/**:
  ros__parameters:
    common:
      target_type: [pointcloud]
      # target_type: [bus, car, motorcycle, trailer, truck, pointcloud]

      on_time_buffer: 0.3
      off_time_buffer: 0.3

      pointcloud:
        range:
          static: false
          dead_zone: 1.0
          forward: 0.0
          backward: 0.0
        height_filter:
          min_height: 0.0
          max_height: 0.0
        voxel_grid_filter:
          x: 0.1
          y: 0.1
          z: 0.5
        clustering:
          cluster_tolerance: 0.2  #[m]
          min_cluster_height: 0.1
          min_cluster_size: 10
          max_cluster_size: 10000
        velocity_estimation:
          observation_time: 0.5

      object:
        range:
          forward: 10.0
          backward: 100.0
        reaction_time: 1.2
        max_velocity: 16.6
        max_deceleration: -2.0

      ego:
        reaction_time: 1.2
        min_velocity: 1.38                              # [m/s]
        max_velocity: 16.6                              # [m/s]
        max_acceleration: 1.5
        max_deceleration: -4.0
        max_positive_jerk: 5.0
        max_negative_jerk: -5.0

      blind_spot:
        active_distance:
          start: 30.0
          end: -5.0

      predicted_path:
        time_resolution: 0.5                            # [s]
        time_horizon: 3.0
        delay_until_departure: 0.0                      # [s]
      rss:
        extended_polygon_policy: along_path             # [-] select "rectangle" or "along_path"
        vehicle_safety_time_margin: 1.0            # [s]
        lateral_distance_max_threshold: 1.8             # [m]
        longitudinal_distance_min_threshold: 3.0        # [m]
        longitudinal_velocity_delta_time: 0.0           # [s]
